TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -190,6 +193,63 @@ def execute(self):
       self.log.error("Could not get the transformations", res['Message'])
     return S_OK()
 
+  def finalize(self):
+    """ Only at finalization: will clean ancient transformations (remnants)
+        1) get the transformation IDs of jobs that are older than 1 year
```

Comment:
Should have a new line before starting the list:

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#enumerated-lists
```suggestion

        1) get the transformation IDs of jobs that are older than 1 year
```